code/modules/shuttles/shuttle.dm

```diff
@@ -126,7 +132,11 @@
 			while (world.time < arrive_time)
 				if(!fwooshed && (arrive_time - world.time) < 100)
 					fwooshed = 1
-					playsound(destination, sound_landing, 100)
+					if(audioalways)
+						playsound(current_location, sound_landing, 100)
+						playsound(destination, sound_landing, 100)
+					else
+						playsound(destination, sound_landing, 100)
```

Comment:
This is actually an oversight, and unrelated. Good catch. 

Though it may be the case that something like this is used in the future it's out of the scope of the PR, so I'll gut it from Long Jump.